src/frontend/komponenter/PersonHeader/PersonHeader.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+
+import styled from 'styled-components';
+
+import { BodyShort, CopyButton, HStack, Heading } from '@navikt/ds-react';
+import { ABorderStrong, ASpacing2, ASpacing4 } from '@navikt/ds-tokens/dist/tokens';
+
+import { usePersonopplysninger } from '../../context/PersonopplysningerContext';
+import { Sticky } from '../Visningskomponenter/Sticky';
+
+const Container = styled(Sticky)`
+    display: flex;
+    align-items: center;
+    gap: ${ASpacing4};
+
+    padding: ${ASpacing2} ${ASpacing4};
+
+    border-bottom: 1px solid ${ABorderStrong};
+    top: 48px;
+`;
+
+const PersonHeader: React.FC = () => {
+    const { personopplysninger } = usePersonopplysninger();
+
+    return (
+        <Container>
+            <Heading size="xsmall">{personopplysninger.navn.visningsnavn}</Heading>
+            <BodyShort>|</BodyShort>
+            <HStack gap="2" align="center">
+                {personopplysninger.personIdent}
+                <CopyButton copyText={personopplysninger.personIdent} size="small" />
+            </HStack>
```

Comment:
Trenger vi egentlige HStack her? Kan man ikke bruke `text` i `CopyButton`?
```suggestion
            <CopyButton
                size="small"
                iconPosition={'right'}
                copyText={personopplysninger.personIdent}
                text={personopplysninger.personIdent}
            />
```

Reply:
Hvis man ikke ønsker at teksten endrer seg når man klikker på den kan man i tillegg bruke `activeText={personopplysninger.personIdent}`

Reply:
Men ser kanskje bedre ut når man bruker Hstack og ikke Text, blir litt annen oppførsel med text

Reply:
Oi, trodde jeg svarte på denne. Hvis man legger tekst inn i copybutton så blir det seende ut som en knapp og ikke som tekst. Holder kanskje at selve kopieringsknappen oppfører seg som en knapp

Reply:
Du kan se forskjellen her på [inline](https://aksel.nav.no/komponenter/core/copybutton#copybuttondemo-inline) og [text](https://aksel.nav.no/komponenter/core/copybutton#copybuttondemo-text)